common/src/main/java/io/github/gaming32/bingo/commandswitch/RepeatableArgumentSwitch.java

```diff
@@ -5,11 +5,11 @@
 import net.minecraft.commands.CommandSourceStack;
 
 import java.util.Collection;
-import java.util.function.Function;
+import java.util.function.BiFunction;
 
 record RepeatableArgumentSwitch<A, T, C>(
     ArgumentSwitch<A, T> inner,
-    Function<Collection<T>, C> collectionFunction
+    BiFunction<CommandContext<CommandSourceStack>, Collection<T>, C> collectionFunction
```

Comment:
Maybe this could be our own functional interface at this point?

Reply:
It's big, but it's only used once, in a package-private class.